module-1/lab-api-scavenger-game/your-code/Challenge1.ipynb

```diff
@@ -1230,2398 +1230,6 @@
     "#Count how many commit objects are there in the array.\n"
```

Comment:
Remember to use relative paths instead of absolute paths!! 
For further occasions, I invite you to normalize your JSON objects before generating the dataframe in order to avoid undesired results.